manager/manager.go

```diff
@@ -346,6 +407,7 @@ func (m *Manager) issue(ctx context.Context, volumeID string) error {
 		if apierrors.IsNotFound(err) {
 			// A NotFound error implies something deleted the resource - fail
 			// early to allow a retry to occur at a later time if needed.
+			m.deletePendingRequestItemForVolume(volumeID)
```

Comment:
Hmm.. is it safe to do this here? What if the CR doesn't exist yet as we haven't yet observed its creation in the lister? Or is this being handled in submitRequest? 👀

Reply:
Make sense. Haven't consider the lister cache catch-up. Will remove this line in next commit.